make_UPD_igv.py

```diff
@@ -0,0 +1,176 @@
+#! /usr/bin/env python3
+import sys
+import argparse
+import vcf
+
+def parse_ped(ped_file):
+    samples = {}  # 'sample_id': {'family': 'fam_id', 'parents': ['sample_id', 'sample_id']}
+    for line in ped_file:
+        ped_data = line.strip().split()
+        family, sample, father, mother, sex, phenotype = ped_data
```

Comment:
if you do this, should you not at least check the len(ped_data) first? (if it has all the columns?)
Or a mini "check pedfile structure" function?

Reply:
Similar to WES-workflow (i.e. Kinship) it is assumed that PED input file is correct.
Incorrect PED file (columns) will crash script with ValueError (not enough values to unpack (expected 6, got x) ) and is therefore likely to be 'caught'